src/main/java/TaskManager.java

```diff
@@ -0,0 +1,134 @@
+public class TaskManager {
```

Comment:
As mentioned previously, it could be better to name it taskLists, but I was wondering if moving this list into the tasks itself as a static variable would have the same effect

Reply:
My rationale for creating a `TaskManager` class was for the purpose of being able to manage my list of `Task`s intuitively in a single interface. Personally, I feel that moving the list into the `Task` class as suggested would not be very intuitively.